backend/src/main/java/app/domain/card/Card.java

```diff
@@ -1,12 +1,46 @@
 package app.domain.card;
 
-public class Card {
+import java.util.Objects;
 
-  private final Rank rank;
-  private final Suit suit;
+public class Card implements Comparable<Card> {
 
-  Card(final Rank rank, final Suit suit) {
-    this.rank = rank;
-    this.suit = suit;
-  }
-}
+	private final Rank rank;
+	private final Suit suit;
+
+	Card(final Rank rank, final Suit suit) {
+		this.rank = rank;
+		this.suit = suit;
+	}
+
+	public Rank getRank() {
+		return this.rank;
+	}
+
+	public Suit getSuit() {
+		return this.suit;
+	}
+
+	private String getCardValue() {
+		return String.join(".", Integer.toString(rank.getValue()), Integer.toString(suit.getValue()));
```

Comment:
Jak potrzebujemy rank value i suit value w stringu to może od razu zrobić je w stringu. Poza tym looknij w angularze w assetsach jak nazywają się pliki z kartami. Nazwy znaczków są literowe, a wartości kart jak dama król są jako Q, K. Byłoby łatwiej się później połapać w logach jakie karty były na stole gdy dostanę info o karcie QH - queen of hearts niż 312

Reply:
to zalezy czy chcemy to pozniej rozwijac o jakas logike ktora zadecyduje kto wygral itd, bo wtedy lepiej int, inaczej moze byc tak jak sugerujesz